.gitignore

```diff
@@ -5,3 +5,4 @@ Haraka
 *~
 bower_components
 coverage
+.idea
```

Comment:
K, no problem. Just curious.

Otherwise +1

On Thu, Mar 24, 2016 at 6:13 PM, Matt Simerson <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/haraka/Haraka/pull/1391#discussion_r57396336>:
>
> > @@ -5,3 +5,4 @@ Haraka
> >  *~
> >  bower_components
> >  coverage
> > +.idea
>
> https://www.jetbrains.com/webstorm/
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/haraka/Haraka/pull/1391/files/46ae731c6e9f80ecd4673a2bf65b362c9f3785e7#r57396336>
>